docs/install_guides/installing-openvino-linux.md

```diff
@@ -30,6 +30,14 @@ The Intel® Distribution of OpenVINO™ toolkit for Linux\*:
 | Additional Tools                                   | A set of tools to work with your models including [Accuracy Checker utility](@ref omz_tools_accuracy_checker_README), [Post-Training Optimization Tool Guide](@ref pot_README), [Model Downloader](@ref omz_tools_downloader_README) and other  |
 | [Documentation for Pre-Trained Models ](@ref omz_models_intel_index)                                   | Documentation for the pre-trained models available in the [Open Model Zoo repo](https://github.com/opencv/open_model_zoo)  |
 
+### Could Be Optionally Installed
+
+Instead of installing the toolkit on your system, you can work with OpenVINO™ components inside the web-based graphical environment of the [OpenVINO™ Deep Learning Workbench](@ref openvino_docs_get_started_get_started_dl_workbench) after a [fast installation from Docker](@ref workbench_docs_Workbench_DG_Docker_Container). <br>
```

Comment:
Not really correct. You probably wanted to say "Instead of using command line tools you can use DL Workbench...".
Or do you refer to the fact that WB docker image has own version of tools installed and you don't need local ones?